src/PriceOracle.sol

```diff
@@ -24,24 +24,34 @@ contract PriceOracle is AccessControl {
     // The minimum price allowed
     uint256 public constant minimumPrice = 1024;
 
+    // The priceBase to modulate the price
+    uint256 public constant priceBase = 514155;
+
     // The current price is the atomic unit.
     uint256 public currentPrice = minimumPrice;
 
     // Constants used to modulate the price, see below usage
-    uint256[] public increaseRate = [0, 1036, 1027, 1025, 1024, 1023, 1021, 1017, 1012];
+    uint256[] public increaseRate = [514191, 514182, 514173, 514164, 514155, 514146, 514137, 514128, 514119];
 
     uint16 targetRedundancy = 4;
     uint16 maxConsideredExtraRedundancy = 4;
 
     // When the contract is paused, price changes are not effective
     bool public isPaused = true;
 
+    // The length of a round in blocks.
+    uint256 public roundLength = 152;
```

Comment:
More detailed description would be appreciated, e.g.: why it is needed, why it is equal to 152, how would increase/decrease affect the code, etc...